src/datetime/tests.rs

```diff
@@ -1477,8 +1477,8 @@ fn locale_decimal_point() {
     assert_eq!(dt.format_localized("%T%.6f", nl_NL).to_string(), "18:58:00,123456");
     assert_eq!(dt.format_localized("%T%.9f", nl_NL).to_string(), "18:58:00,123456780");
 
-    assert_eq!(dt.format_localized("%T%.f", ar_SY).to_string(), "18:58:00.123456780");
-    assert_eq!(dt.format_localized("%T%.3f", ar_SY).to_string(), "18:58:00.123");
-    assert_eq!(dt.format_localized("%T%.6f", ar_SY).to_string(), "18:58:00.123456");
-    assert_eq!(dt.format_localized("%T%.9f", ar_SY).to_string(), "18:58:00.123456780");
+    assert_eq!(dt.format_localized("%T%.f", ar_SY).to_string(), "18:58:00٫123456780");
```

Comment:
An Arabic comma (or what is it called) in roman numerals doesn't seem right.

Reply:
Agreed, some locales have `outdigits` defined as part of `LC_CTYPE`, but `pure-rust-locales` doesn't support these. I did check though and there's nothing defined for `ar_SY`. I beleive that ICU has the numbers, though I'm still researching.